src/shogun/mathematics/graph/CMakeLists.txt

```diff
@@ -90,8 +95,13 @@ IF (ENABLE_TESTING)
 	set(SHOGUN_TEST_LINK_LIBS shogun_test_main shogun-engine)
 
 	ADD_SHOGUN_TEST(nodes/Add_test)
+	ADD_SHOGUN_TEST(nodes/Subtract_test)
+
 	if (UNIX AND NOT APPLE)
 		set_property(TEST Add_test PROPERTY ENVIRONMENT "LD_LIBRARY_PATH=${PROJECT_BINARY_DIR}/src/shogun")
+		set_property(TEST Subtract_test PROPERTY ENVIRONMENT "LD_LIBRARY_PATH=${PROJECT_BINARY_DIR}/src/shogun")
 	endif()
 	add_dependencies(Add_test ${EXCUTORS})
+	add_dependencies(Subtract_test ${EXCUTORS})
+
```

Comment:
@vigsterkr I am guessing this block could become a function/macro?

Reply:
yep same as all the library setter magic for abi version etc.